clients/search-component/src/TrieveModal/index.tsx

```diff
@@ -20,6 +20,15 @@ const Modal = () => {
 
   const ButtonEl = props.ButtonEl;
 
+  const [isMobile, setIsMobile] = useState(false);
+
+  useEffect(() => {
+    const checkWindowSize = () => setIsMobile(window.innerWidth <= 768);
+    window.addEventListener("resize", checkWindowSize);
+    checkWindowSize();
+    return () => window.removeEventListener("resize", checkWindowSize);
+  }, []);
```

Comment:
We can use tailwind classes for this, https://tailwindcss.com/docs/responsive-design